webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  plugins: [
+    new CopyPlugin([
+      { from: 'img', to: 'img' },
+      { from: 'manifest.json' },
+      { from: 'options', to: 'options' },
+      { from: 'background', to: 'background' },
+      { from: 'just-not-sorry.css' },
+    ]),
+  ],
+  mode: 'production',
+  entry: './src/JustNotSorry.js',
+  output: {
+    path: path.join(__dirname, 'build'),
```

Comment:
Since the bundled code for the extension is now all placed in the build directory, the `package.sh` script need to be updated so that it zips up the contents of the build directory.  You can probably replace include.lst with a simple wildcard, but you'd need to also copy the readme and privacy policy to the build directory.

Reply:
The script in `package.sh` has been updated to use the build files and the `README.md` instructions have also been updated to include instructions for generating the build.